samples/arcgis-widget/src/components/Daylight/Daylight.tsx

```diff
@@ -1,9 +1,12 @@
 import React, { useCallback, useState } from "react";
+import { generateId } from "@vertigis/arcgis-extensions/Entity";
```

Comment:
Although lots of our code imports this utility from here, in general I would prefer to use generateUuid() from `@vertigis/arcgis-extensions/utilities/uuid`, especially when the use case is unrelated to entities.

Reply:
I found a number used in our code base. Thanks for pointing me at the right one 😛